ERP/app/Http/Kernel.php

```diff
@@ -55,8 +55,6 @@ class Kernel extends HttpKernel
     protected $routeMiddleware = [
         'inventory.access.only' => \App\Http\Middleware\InventoryAccessOnly::class,
         'manufacturer.access.only' => \App\Http\Middleware\ManufacturerAccessOnly::class,
-        'accountant.access.only' => \App\Http\Middleware\AccountantAccessOnly::class,
-        'sales.access.only' => \App\Http\Middleware\SalesAccessOnly::class,
```

Comment:
Remember to pull from develop before you create your new branch, you're deleting these that you introduced